app/src/main/java/hr/foi/air/sportloc/MainActivity.java

```diff
@@ -8,6 +8,6 @@
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
-        setContentView(R.layout.activity_main);
+        setContentView(R.layout.activity_registration);
```

Comment:
This should be for reason stated before
```suggestion
        setContentView(R.layout.activity_main);
```